### PR TITLE
fdk-aac

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: mingw-w64-fdk-aac/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+_realname=fdk-aac
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=5fd7e6537d1f37fb1bec8c6aa2f8e22030ab10de
```
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: Repository have normal versions. If you want latest then create PKGBUILD for git version

**Comment by Alexpux** [comment]
I will close pull request if my comment not fixed.

**Comment by galeksandrp** [comment]
Like this?

File: mingw-w64-fdk-aac/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,31 @@
+_realname=fdk-aac-git
```
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: Add header

File: mingw-w64-fdk-aac/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=5fd7e6537d1f37fb1bec8c6aa2f8e22030ab10de
+pkgrel=1
+pkgdesc='A standalone library of the Fraunhofer FDK AAC code from Android'
+arch=('any')
+license=('custom')
+makedepends=('autoconf' 'automake' 'libtool' 'make')
+url="https://github.com/mstorsjo/${_realname}"
+source=("https://github.com/mstorsjo/${_realname}/archive/${pkgver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: Download in file with name and version

**Comment by mati865** [comment]
Rename directory to ` mingw-w64-fdk-aac-git`.

File: mingw-w64-fdk-aac/PKGBUILD (hunk 3 above)
Comment (mati865) [inline mingw-w64-fdk-aac/PKGBUILD]: @galeksandrp look here https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-angleproject-git/PKGBUILD#L6

File: mingw-w64-fdk-aac/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+sha256sums=('SKIP')
```
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: Run updppkgsums to generate proper checksum

File: mingw-w64-fdk-aac/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+prepare() {
+	cd ${srcdir}/${_realname}-${pkgver}
+
+	./autogen.sh
+	./configure --prefix=${MINGW_PREFIX}
```
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: move configure into build

File: mingw-w64-fdk-aac/PKGBUILD (hunk 2 above)
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: And remove git from package name

File: mingw-w64-fdk-aac/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,31 @@
+_realname=fdk-aac-git
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=5fd7e6537d1f37fb1bec8c6aa2f8e22030ab10de
```
Comment (galeksandrp) [inline mingw-w64-fdk-aac/PKGBUILD]: It fails to build.

**Comment by Alexpux** [comment]
I'm still not understand what package is adding - release or git, because of mixing things in PKGBUILD. If it will be git package it will not be builded for repo in any case. I'm stop building newly adding git packages for everything.

File: mingw-w64-fdk-aac/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,34 @@
+_realname=fdk-aac
```
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: Add header.

File: mingw-w64-fdk-aac/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}-git")
```
Comment (Alexpux) [inline mingw-w64-fdk-aac/PKGBUILD]: This is not git package

File: mingw-w64-fdk-aac/PKGBUILD (hunk 7 above)
Comment (galeksandrp) [inline mingw-w64-fdk-aac/PKGBUILD]: Which header?

**Comment by Alexpux** [comment]
Import patch in repo instead downloading it.

**Comment by galeksandrp** [comment]
Done.

**Pull request merged**